pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -60,6 +63,28 @@ def __init__(
         self.df_whitelist = load_data.load_whitelist()
         self.df_whitelist = self.df_whitelist[self.df_whitelist["inference_ok"] == True]
 
+    @staticmethod
+    def _get_county_hospitalization_from_actuals(fips, t0_simulation):
```

Comment:
This method name is misleading. If a fips has cumulative, but not current, this will derive an estimate of current that is not an 'actual' from the datasources.

Reply:
Updated, added a doc string too.